tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -256,9 +263,12 @@ version of the raw dataset will probably go away.
       <urban_filename>lnd/clm2/rawdata/gao_oneill_urban/historical/urban_properties_GaoOneil_05deg_ThreeClass_1850_cdf5_c20220910.nc</urban_filename>
     </entry>
 
-    <!-- Historical period from 1850 to 2015 -->
-    <entry pft_years="1850-2015" >
-      <data_filename>lnd/clm2/rawdata/pftcftdynharv.0.25x0.25.LUH2.histsimyr1850-2015.c20230226/mksrf_landuse_ctsm52_histLUH2_%y.c20230226.nc</data_filename>
+    <!-- Historical period from 1700 to 2022 (expect 2023 soon) -->
+    <!-- Lake and urban data from 1700 to 1849 is copied from 1850 -->
+    <!-- Lake data from 2018 to 2023 is copied from 2017 -->
+    <!-- Urban data from 2016 to 2023 is copied from 2015 -->
+    <entry pft_years="1700-2022" >
```

Comment:
@olyson I wanted to let you know how I handled this extension of our definition of historical.

As explained in the comment above,
- I copied the 1850 urban all the way back to 1700 (same for lake) and
- I copied the 2015 urban out to 2023.

Let me know if you have concerns/comments/suggestions about this.

Reply:
Just thinking through this...the historical data starts changing PCT_URBAN in 2001, so changes from 2001-2015 will be maintained with your approach.  And then we'll have 2015 urban for 2015-2023.   My understanding from the CLM meeting is that we'll only support the CMIP6 version of the SSPs in land-only simulations, which will use the GaoOneill data.  In those simulations then, there will be a bit of a discontinuity between 2023 and 2024 as we transition from historical to SSP data.  Which I guess would be ok.  Is my thinking correct here?
Now I'm wondering about what to do about the CMIP7 version of the SSPs or whatever they will be called.  Will they be close enough to CMIP6 definitions such that we can continue to use the GaoOneill data, or are we going to need to  derive new PCT_URBAN trajectories from, e.g., the LU transition matrices?

Reply:
> Just thinking through this...the historical data starts changing PCT_URBAN in 2001, so changes from 2001-2015 will be maintained with your approach. And then we'll have 2015 urban for 2015-2023. My understanding from the CLM meeting is that we'll only support the CMIP6 version of the SSPs in land-only simulations, which will use the GaoOneill data. In those simulations then, there will be a bit of a discontinuity between 2023 and 2024 as we transition from historical to SSP data. Which I guess would be ok. Is my thinking correct here?

That's my understanding, as well. Similar to the small discontinuity that we will see in landuse data, as @lawrencepj1 pointed out.

Reply:
No further concerns on this, so I'm resolving.